controller/work_item_link_type.go

```diff
@@ -89,8 +91,7 @@ func enrichLinkTypeList(ctx *workItemLinkContext, list *app.WorkItemLinkTypeList
 	for categoryID := range categoryIDMap {
 		modelCategory, err := ctx.Application.WorkItemLinkCategories().Load(ctx.Context, categoryID)
 		if err != nil {
-			jerrors, httpStatusCode := jsonapi.ErrorToJSONAPIErrors(err)
-			return ctx.ResponseData.Service.Send(ctx.Context, httpStatusCode, jerrors)
+			err
```

Comment:
I am fine with handling the error in the calling controller. If so then remember that `workitemtype.go` controller also use it. So you have to adapt the code there too.

On the other hand, let's be explicit and write `return err` like in the rest of the code. Other people might get lost when you use this type of return in golang :).